### ISSUE TITLE
Add options in summary menu

### ISSUE THREAD

**Issue by Kirirru** [lairdubois/lairdubois-opencutlist-sketchup-extension#659]
Please add the ability to print all drawings and all cutting diagrams at once. So that you don't have to go in and print each position individually. 
![diag](https://github.com/user-attachments/assets/ab3776d8-a85b-413f-99ab-a95d5f43c7d2)

**Comment by Kirirru** [comment]
a small logical Mistake in the translation into Russian. 
![Снимок экрана 2024-09-17 211950](https://github.com/user-attachments/assets/aa9f7dfd-1b7b-4ad1-b23c-7a55a5135759)
It's more Fitting here - "with" - "Есть" "without" - "Нет"

**Comment by bbeaulant** [comment]
Hello @Kirirru,

Drawing can already be called for the whole project by clicking on the top bar "Drawing" button.

Cutting diagrams and Labels can't be generated from a single place because it represent multiple configuration context.
I know that it could be easier, but quite impossible to keep a simple way to integrate this in a fluid UI/UX process.

Thank you for reporting the typo.

**Comment by Kirirru** [comment]
I'm not insisting, I'm just trying to understand. Why can't you use the settings that have already been configured for each type or the default settings for each type? I mean to automate the functionality of entering each position with subsequent printing. The fact is that the calculation is made more than once. Sometimes the number of parts varies. But the position has already been configured. And before starting the work, you need to go into each of the 10-20 positions to print a drawing and diagram.

**Comment by Kirirru** [comment]
Generating all cutting diagrams in background is already done in Estimate module.  What? Where?

**Comment by Kirirru** [comment]
What if you don't use the options window? Only the functionality of the "generate" button in the "cutting diagram" menu. I do not understand why the information of several HTML cannot be placed in one.

**Comment by bbeaulant** [comment]
Because everything on the cutting diagram screen is designed to represent only **one** material / std size pair.

Displaying multiple materials on the same screen break this simple logic ... so as I said, UI will be complex for nothing.

I understand you need. But I don't want to create a gas machine for this.

**Comment by Kirirru** [comment]
Now i understand. Thank you anyway.
